packages/Graph/Actions/User/GetDirectReports.cs

```diff
@@ -41,6 +41,12 @@ public class GetDirectReports : BaseMsGraphCustomAction<IEnumerable<DirectoryObj
         [JsonProperty("MaxCount")]
         public IntExpression MaxCount { get; set; }
 
+        /// <summary>
+        /// Gets or sets the fields to select from the Graph API.
+        /// </summary>
+        [JsonProperty("FieldsToSelect")]
+        public StringExpression FieldsToSelect { get; set; }
```

Comment:
Should this not really be an ArrayExpression? And then do a join on the string values when passing through as a query parameter on the request itself?